README.md

```diff
@@ -31,11 +31,13 @@ Table of Contents
 
 ## Get Involved
 
-There is a lot to do! If you are interested in getting involved, please join
-the mailing lists and attend the community meetings. If you're interested in
-contributing to a specific part of the project, please join the appropriate
-special interest group (SIG). Details for all of these items are below. We are
-a friendly, collaborative group and look forward to working together!
+There is a lot to do! If you are interested in getting involved, here's how you can start:
+
+1. Check out our [contributor guide](https://github.com/open-telemetry/community/tree/main/guides/contributor) to learn about our contribution process.
```

Comment:
```suggestion
1. Check out our [contributor guide](./guides/contributor) to learn about our contribution process.
```

(we don't do tags or branches in this repo but if one is reviewing a PR it would take them to the wrong place otherwise)